pkg/errors/errors.go

```diff
@@ -74,7 +74,7 @@ func NewPullImageError(name string, err error) error {
 		Message:    fmt.Sprintf("unable to get %s", name),
 		Details:    err,
 		ErrorCode:  PullImageError,
-		Suggestion: "check image name, or if using local image add --pull-policy=never flag",
+		Suggestion: fmt.Sprintf("check image name, or if using a local image set the builder image pull policy to %q", "never"),
```

Comment:
is this preferred over escaping quotes in the string? Seems odd.

Reply:
%q seems to be a short cut, and I've seen it used elsewhere. Under the hood go does additional cleanup, like trimming whitespace.
If this was a performance-intensive, I wouldn't use it (then again, I'd avoid `fmt` for anything high-performance)

Reply:
k